students/rfelts/lesson03/list_lab.py

```diff
@@ -1,2 +1,139 @@
 #!/usr/bin/env python3
+# Russell Felts
+# Assignment 3 - List Lab Exercise
 
+
+# Series 1
+
+def series1():
+    """
+    Create a list and modify it by:
+    Asking the user for another fruit and adding it to the end of the list
+    Adding another fruit to the beginning of the list using “+”
+    Adding another fruit to the beginning of the list using insert().
+    Also print out the every item in the list that start with the letter P.
+
+    Parameters:
+    arg1 (list) The list to modify
+
+    Returns:
+    list: the modified list
+    """
+    print("Start series 1...\n")
+
+    fruit_list = ["Apples", "Pears", "Oranges", "Peaches"]
+    fruit_list_copy = fruit_list[:]
+
+    print("List:", fruit_list, "\n")
+    response = input("Please enter another fruit: ")
+    fruit_list.append(response)
+    print(fruit_list, "\n")
+
+    response = input("Please enter a number: ")
+    # Validate the user input
+    while int(response) > len(fruit_list) or int(response) <= 0:
+        print("\nThe number entered is not on the list range.\n")
+        response = input("Please enter a number: ")
+
+    print("\nThe number " + str(response) + " item in the list is: " +
+          fruit_list[int(response) - 1] + "\n")
+
+    fruit_list = ["Pineapples"] + fruit_list[:]
+    print(fruit_list, "\n")
+
+    fruit_list.insert(0,"Cherries")
+    print(fruit_list, "\n")
+
+    #Display all the fruits that begin with “P”
+    print("Printing all fruits in the list that start with the letter P:\n")
+    for x in fruit_list:
+        if x[:1] == "P":
```

Comment:
a more common convention is to check on zero element -> `if x[0] == "P"`
or
`if x.startswith("P"):`